.circleci/config.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           command: |
             mkdir -p ~/test-reports
             echo ${_RUBY_VERSION} > .ruby-version
-            brew untap caskroom/versions
+            brew untap caskroom/versions || true
```

Comment:
macos users: is it okay to fix it so?
I han an issue in ci first time:
https://circleci.com/gh/fastlane/fastlane/40777

Reply:
Seems fine to me 🙃